src/index.scss

```diff
@@ -52,10 +52,11 @@ $white: #fff;
   }
   .logo {
     display: block;
-    align-self: flex-end;
     box-sizing: content-box;
-    height: 2.5rem;
-    padding: .5rem .5rem 1.35rem;
+    position: relative;
+    top: -.05em;
```

Comment:
Booo negative positioning!

Reply:
This the best way to visually balance the edx logo vertically. I'm using ems and positional relative so this doesn't affect layout